scripts/load_inventory_demo.py

```
@@ -0,0 +1,387 @@
+        Initializes the inventory demo loader.
+
+        :param orders_count: The total number of orders to create.
+        :param paid_ratio: The fraction of orders that are marked paid.
+        :param seed: The seed for the pseudo random generator, so that
+        successive runs produce a deterministic data set.
+        :param verbose: Whether to print verbose output.
+        """
+
+        self.orders_count = orders_count
+        self.paid_ratio = paid_ratio
+        self.random = Random(seed)
```

Ruff (0.15.11)

[error] Standard pseudo-random generators are not suitable for cryptographic purposes

(S311)